### PR TITLE
Silu simpler

### PR DESCRIPTION
**Pull request by mattdangerw**

### CODE REVIEW

**Comment by mattdangerw** [comment]
@tirthasheshpatel does this make sense to you? Seems best to rely on the torch op directly if we can.

**Review by tirthasheshpatel — APPROVED** [review]
Looks good, thanks for the fix @mattdangerw!

File: keras/backend/torch/nn.py [hunk 1]
```diff
@@ -47,9 +47,9 @@ def softsign(x):
     return tnn.softsign(x)
 
 
-def silu(x, beta=1.0):
+def silu(x):
     x = convert_to_tensor(x)
-    return x * sigmoid(beta * x)
```
[the remark this answers is not in the document]
Comment (mattdangerw) [reply keras/backend/torch/nn.py:52]: Do we want to expose it? The weird part was that this didn't exist on the actual `keras.ops` signature or jax or numpy.

That backend should match the `ops` signature right? We don't want secret extra options handing around on specific backends.

[the remark this answers is not in the document]
Comment (fchollet) [reply keras/backend/torch/nn.py:52]: If it's not in the `op` signature we can drop it.

**Review by fchollet — APPROVED** [review]

**Pull request merged**